src/backend/core/services/opensearch.py

```diff
@@ -267,22 +280,102 @@ def ensure_index_exists(index_name):
         opensearch_client().indices.create(
             index=index_name,
             body={
-                "settings": {"index.knn": True},
+                "settings": {
```

Comment:
I think we should create a package for the language settings, only the french one for now. 
I assume that we will have to create an index for each supported language later : can be donne with a suffix to  the index_name and a new "lang" argument for the `ensure_index_exists` function.